src/chrome/content/rules/Caller.com.xml

```diff
@@ -53,24 +53,18 @@ Fetch error: http://caller.com/ => https://www.caller.com/: (51, "SSL: no altern
 <ruleset name="Caller.com (partial)" default_off='failed ruleset test'>
 
 	<target host="caller.com" />
+	<target host="www.caller.com" />
+	<target host="login.caller.com" />
 	<target host="*.caller.com" />
 
 
 	<securecookie host="^(?:login)?\.caller\.com$" name=".+" />
 
 
-	<rule from="^http://(?:www\.)?caller\.com/"
+	<rule from="^http://caller\.com/"
```

Comment:
I get an invalid name here (and connection refused @ https://caller.com).

Perhaps limit this ruleset to login.caller.com only?